### PR TITLE
Clean up JS files

### PR DESCRIPTION
**Pull request by JoshuaLuckers**
### What does it do?
Remove unnecessary comments and uncommented code in JS files and remove some extra whitespaces at some places.

### Why is it needed?
Currently contributors remove unnecessary comments and uncommented code when they encounter it when fixing/changing the file. This is great and but creates unnecessary noise and/or make it difficult to see actual changes in a PR.

Doing it all at once cleans up the code base and reduces the noise. 

### Related issue(s)/PR(s)
n/a

### CODE REVIEW

**Comment by JoshuaLuckers** [comment]
Please let me know if I missed any!

**Comment by opengeek** [comment]
I like this, but isn't this going to conflict with every outstanding PR that affects JS?

**Comment by JoshuaLuckers** [comment]
> I like this, but isn't this going to conflict with every outstanding PR that affects JS?

Most likely yeah. A small annoyance for contributors affected by it but in the long term both contributors and reviewers will benefit from it. By doing it once we have less noise in other PR's and make diffs easier to read.

**Comment by JoshuaLuckers** [comment]
I also wouldn't mind if the other existing PRs would go first and that I resolve the conflicts.

**Comment by Ruslan-Aleev** [comment]
Great!
It would be useful, before the official release of 3.x, to implement this PR too - https://github.com/modxcms/revolution/issues/14311

File: manager/assets/modext/core/modx.js [hunk 1]
```diff
@@ -457,8 +457,7 @@ Ext.extend(MODx,Ext.Component,{
 					}
 				}]
 			}]
-			//,html: '<iframe src="' + url + '" width="100%" height="100%" frameborder="0"></iframe>'
-        });
+        });w
```
Comment (theboxer) [inline manager/assets/modext/core/modx.js]: Seems like the `w` shouldn't be there :)

**Review by theboxer — CHANGES_REQUESTED** [review]

**Comment by theboxer** [comment]
@opengeek I wouldn't be too concerned about conflicts, as all changes are only around removing comments and new lines here and there, so all conflicts (if they occur) should be fairly simple to resolve.

Good job @JoshuaLuckers

**Review by theboxer — APPROVED** [review]

**Review by Ruslan-Aleev — APPROVED** [review]

**Comment by opengeek** [comment]
@JoshuaLuckers — why did we close this?

**Comment by opengeek** [comment]
@JoshuaLuckers — did this just get merged in a way that did not show it as merged? I'm confused as to what happened here.

**Comment by JoshuaLuckers** [comment]
@opengeek yes, I did a squash merge but GitHub didn't recognize it for some reason..

**Comment by opengeek** [comment]
Ok, yeah, if you squash it yourself, you have to force push it to the PR branch first. Squash only works with the buttons here in the UI otherwise.

**Comment by JoshuaLuckers** [comment]
Good to know, thanks!

**Pull request closed without merging**